ha/k8s_setup/ha_setup.py

```diff
@@ -276,6 +276,7 @@ def process(self):
 
             Log.info('Creating cluster cardinality')
             self._confStoreAPI = ConftStoreSearch()
+            self._confStoreAPI.get_control_pod(self._index)
```

Comment:
You are returning machine ids but not capturing here.

Reply:
This call was made just for testing purpose, to test fetched the machine-id.
Removed this call from updated patch

Reply:
Ok..Thanks!